schema/schema.sql

```diff
@@ -0,0 +1,73 @@
+-- Creating a Lable Table
+CREATE TABLE "Label" (
+  "id" bigserial,
+  "title" varchar(150),
+  "color" varchar(100),
+  PRIMARY KEY ("id")
+);
+
+-- Creating a Book Table
+CREATE TABLE "Book" (
+  "id" bigserial,
+  "genre_id" int,
+  "author_id" int,
+  "label_id" int,
+  "publish_date" date,
+  "archived" bool,
+  "publisher" varchar(200),
+  "cover_state" varchar(10),
+  PRIMARY KEY ("id"),
+  CONSTRAINT "FK_Book.label_id"
+    FOREIGN KEY ("label_id")
+      REFERENCES "Label"("id")
+);
```

Comment:
- [ ] Kindly create the remaining two foreign keys for the `Book` table. Remember the `Book` inherits from the `Item` class. Therefore all foreign keys that apply to the `Item` class apply to the `Book` class. 👍

Reply:
Hello @Gambit142 👋️, thank you for the feedback. We will implement the changes🤗️